src/Math-Matrix/PMMatrix.class.st

```diff
@@ -67,6 +67,11 @@ a tr.
 "15"
 ]
 
+{ #category : #'instance creation' }
+PMMatrix class >> identity: anInteger [ 
+  ^ PMSymmetricMatrix identity: anInteger 
+]
+
```

Comment:
What is anInteger? Is that the matrix size? If so, consider improving that argument name.

Reply:
Would `dimension` be a better variable name instead of `anInteger`?

Reply:
@rakki-18 yes, great idea :+1:

Reply:
Identity matrices are usually square matrices.if we have a square matrix class already, should it moved there?

Reply:
If we do not have the idea of Square Matrices, then maybe, instead of `initialize` we should have a class level factory method named `squareWith: dimension`

Reply:
I understand that PMMatrix represents square AND non-square matrices.since the identity matrix is square. It maybe safe here for the time being.